test.py

```diff
@@ -0,0 +1,83 @@
+import unittest
+from dojo import Dojo
+
+class TestCreateRoom(unittest.TestCase):
+    """Test cases for creating room"""
+    def setUp(self):
+        self.dojo = Dojo()
+
+    def test_adds_office_successfully(self):
+        """
+        Tests if room of type office can be added successfully
+        it should the increase the number of offices.
+
+        """
+        current_no_offices = len(self.dojo.offices)
+        office = self.dojo.create_room("office", "A04")
+        new_no_offices = len(self.dojo.offices)
+        self.assertEqual(office, "Office added successfully")
+        self.assertEqual(new_no_offices, current_no_offices+1)
```

Comment:
change the syntax of this assertion to reflect the change I indicated as you did in lines 72 and 65